C:\Users\ctw00071\Desktop\Desktop\Books\TestFunctions-unittest\code\ch1>pytest test_one.py
========================================================================================================= test session starts =========================================================================================================
platform win32 -- Python 3.7.0, pytest-4.2.1, py-1.7.0, pluggy-0.8.1
rootdir: C:\Users\ctw00071\Desktop\Desktop\Books\TestFunctions-unittest\code\ch1, inifile:
collected 1 item                                                                                                                                                                                                                       

test_one.py .                                                                                                                                                                                                                    [100%]

====================================================================================================== 1 passed in 0.01 seconds =======================================================================================================

C:\Users\ctw00071\Desktop\Desktop\Books\TestFunctions-unittest\code\ch1>pytest -v test_one.py
========================================================================================================= test session starts =========================================================================================================
platform win32 -- Python 3.7.0, pytest-4.2.1, py-1.7.0, pluggy-0.8.1 -- c:\users\ctw00071\appdata\local\programs\python\python37\python.exe
cachedir: .pytest_cache
rootdir: C:\Users\ctw00071\Desktop\Desktop\Books\TestFunctions-unittest\code\ch1, inifile:
collected 1 item                                                                                                                                                                                                                       

test_one.py::test_passing PASSED                                                                                                                                                                                                 [100%]

====================================================================================================== 1 passed in 0.01 seconds =======================================================================================================

C:\Users\ctw00071\Desktop\Desktop\Books\TestFunctions-unittest\code\ch1>pytest test_two.py
========================================================================================================= test session starts =========================================================================================================
platform win32 -- Python 3.7.0, pytest-4.2.1, py-1.7.0, pluggy-0.8.1
rootdir: C:\Users\ctw00071\Desktop\Desktop\Books\TestFunctions-unittest\code\ch1, inifile:
collected 1 item                                                                                                                                                                                                                       

test_two.py F                                                                                                                                                                                                                    [100%]

============================================================================================================== FAILURES ===============================================================================================================
____________________________________________________________________________________________________________ test_failing _____________________________________________________________________________________________________________

    def test_failing():
>       assert (1, 2, 3) == (3, 2, 1)
E       assert (1, 2, 3) == (3, 2, 1)
E         At index 0 diff: 1 != 3
E         Use -v to get the full diff

test_two.py:2: AssertionError
====================================================================================================== 1 failed in 0.04 seconds =======================================================================================================

C:\Users\ctw00071\Desktop\Desktop\Books\TestFunctions-unittest\code\ch1>pytest -v test_two.py
========================================================================================================= test session starts =========================================================================================================
platform win32 -- Python 3.7.0, pytest-4.2.1, py-1.7.0, pluggy-0.8.1 -- c:\users\ctw00071\appdata\local\programs\python\python37\python.exe
cachedir: .pytest_cache
rootdir: C:\Users\ctw00071\Desktop\Desktop\Books\TestFunctions-unittest\code\ch1, inifile:
collected 1 item                                                                                                                                                                                                                       

test_two.py::test_failing FAILED                                                                                                                                                                                                 [100%]

============================================================================================================== FAILURES ===============================================================================================================
____________________________________________________________________________________________________________ test_failing _____________________________________________________________________________________________________________

    def test_failing():
>       assert (1, 2, 3) == (3, 2, 1)
E       assert (1, 2, 3) == (3, 2, 1)
E         At index 0 diff: 1 != 3
E         Full diff:
E         - (1, 2, 3)
E         ?  ^     ^
E         + (3, 2, 1)
E         ?  ^     ^

test_two.py:2: AssertionError
====================================================================================================== 1 failed in 0.03 seconds =======================================================================================================

C:\Users\ctw00071\Desktop\Desktop\Books\TestFunctions-unittest\code\ch1>
